client/src/components/edit-user-component/index.jsx

```diff
@@ -9,46 +11,37 @@ import {
   Typography,
 } from "@mui/material";
 import { Form, Header, Cancel } from "./index.styles";
+import ConfirmDialog from "../confirm-dialog-component";
+
+function EditComponent({ userID, onModifyUser }) {
+  const navigate = useNavigate();
 
-function EditComponent() {
   const [firstName, setFirstName] = useState("");
   const [lastName, setLastName] = useState("");
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
+  const [confirmEditDialog, setConfirmEditDialog] = useState(false);
 
   const handleSave = (e) => {
     e.preventDefault();
-    // Create a JSON object with the required keys and values
+    setConfirmEditDialog(true);
+  };
+
+  const confirmEdit = () => {
     const userData = {
+      userID,
       firstName,
       lastName,
       email,
-      password,
+      isAdmin: false,
     };
+    onModifyUser(userData);
+    navigate("/admin");
+    setConfirmEditDialog(false);
+  };
 
-    // Convert the JSON object to a string
-    const userDataJSON = JSON.stringify(userData);
-
-    // Replace url with target route
-    fetch("http://localhost:8000/edit", {
```

Comment:
If we remove this, will we have a spot where the user data will be accessed to pre-populate the fields for the editing of a user?

Reply:
Hi Jordan, the code that I removed from this file was there before I touched it, we don't have an end point to edit a user with connect to the backend yet so there was compiling issues when we enter the page. So I left it out because it would just lead to an error when we click the submit button. When we have the end points for edit user, I will be able to link everything up again.